### ISSUE TITLE
Trying to delete items on server.

### ISSUE THREAD

**Issue by Chappy130** [Funwayguy/RealisticItemDrops#20]
I have been trying to clear all dropped items on my server I am running, as one of my players dropped too many items from a sieve and now whenever he logs on it crashes the server. How can I clear all the items.

**Comment by truehunterethan** [comment]
yeah for some reason /kill @e[type=item] doesn't work for me. but you can try it.

**Comment by Funwayguy** [comment]
I think I'm just going to roll back all the changes I made to the last few builds to fix mods double checking their drops weren't deleted/replaced. There are so many other issues people are having with it and I'm frankly getting tired of it.

Note: This won't fix the kill command because the entity has a completely different ID `realdrops:item_loot`

**Comment by Alynthia** [comment]
After a bit of tinkering, found the solution:
`/kill @e[type=realdrops:item_loot]`
